### ISSUE TITLE
Font

### ISSUE THREAD

**Issue by myzinsky** [cdeschryver/tukl-eit-report#3]
for the CI shouldn't be there this font (at least for headings):

http://www.uni-kl.de/universitaet/pr-marketing/marketing/cd-und-vorlagen/schriftarten-pt-sans/

**Comment by cdeschryver** [comment]
Jep, should be. Therefore I've included the following:
```
% PTSans - corporate design font of the University of Kaiserslautern
\usepackage{paratype} 
```
Isn't that working for you?

**Comment by myzinsky** [comment]
I Think that font is different right:

<img width="475" alt="bildschirmfoto 2016-10-06 um 09 30 43" src="https://cloud.githubusercontent.com/assets/1970554/19154210/908a2432-8ba7-11e6-9790-b43c49291c93.png">\

It should look like the fonts in the TU Logo right?

**Comment by myzinsky** [comment]
I think what you should do is:

``` latex
\usefont{T1}{PTSansCaption-TLF}{m}{it} TEST HEADING \\
```

You can assign this issue to me and I will do a pull request.

**Comment by cdeschryver** [comment]
You're right. Just fixed that in 7ef1ab6b5565e2bfac16af18453ccc1527659fbc . Thx.
